apps/server/internal/runner/client_branch_coverage_test.go

```
@@ -0,0 +1,272 @@
+	server := newProtocolTestServer(t, requiredRunnerCapabilities(), func(conn *websocket.Conn, msg protocol.Message) {
+		received <- msg
+		if msg.Type == protocol.TypeStart {
+			writeProtocol(t, conn, protocol.TypeSessionStarted, msg.SessionID, nil)
+		}
+	})
+	defer server.Close()
+	conn, err := Dial(context.Background(), wsURL(server.URL))
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer conn.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)